src/util.ml

```diff
@@ -263,6 +272,9 @@ value gen_decline2 wt s1 s2 =
   (*surtout pas ! Translate.eval*) (loop 0)
 ;
 
+value transl_a_to_b conf x y =
+  gen_decline2 (transl_nth conf "%1 to %2" 0) x y
```

Comment:
Adding this new delimiter `|` did you see that since it uses the same function `gen_decline` you may broke some translation ? What about a text that contains a `|`, e.g. `"his|her"`